boards/arm/stm32f411e_disco/doc/stm32f411e_disco.rst

```diff
@@ -0,0 +1,184 @@
+.. _stm32f411e_disco_board:
+
+STM32F411E-DISCO
+################
+
+Overview
+********
+
+The STM32F411E Discovery kit features an ARM Cortex-M4 based STM32F411VE MCU with a wide range of connectivity support and configurations Here are some highlights of the STM32F411E-DISCO board:
```

Comment:
Should break this into lines < 80 chars please.  Add a period:

    and configurations. Here are

Reply:
OK, updated. Thank you!